### PR TITLE
Add image builders to Examples

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: examples/kubeflow_pipelines_orchestration/README.md [hunk 1]
```diff
@@ -270,7 +270,9 @@ in Kubeflow Pipelines. We need to configure it with the right kubernetes
 * context so ZenML can run pipelines in your GCP cluster.
 * Kubectl can [access](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) 
 your GCP Kubernetes cluster.
-
+* An ***Image Builder** component that is responsible for building docker images
```
Comment (strickvl) [inline examples/kubeflow_pipelines_orchestration/README.md:273]:
```suggestion
* An ***Image Builder** component that is responsible for building Docker images
```

File: examples/kubernetes_orchestration/README.md [hunk 2]
```diff
@@ -128,6 +128,16 @@ zenml artifact-store register s3_store
     --path=<REMOTE_ARTIFACT_STORE_PATH>
 ```
 
+#### Setup Image Builder
+
+In order to build the Docker images for the Kubernetes pods, we will use the
+`docker` image builder.
```
Comment (strickvl) [inline examples/kubernetes_orchestration/README.md:134]:
```suggestion
`local` image builder.
```

File: examples/spark_distributed_programming/README.md [hunk 3]
```diff
@@ -265,6 +265,16 @@ zenml container-registry register spark_container_registry \
     --uri=$ECR_URI
 ```
 
+We also need to register a **image builder** which will be used to build the
```
Comment (strickvl) [inline examples/spark_distributed_programming/README.md:268]:
```suggestion
We also need to register an **image builder** which will be used to build the
```

File: examples/step_operator_remote_training/README.md [hunk 4]
```diff
@@ -67,6 +67,8 @@ The stack will consist of:
   artifacts of your pipeline.
 * The **Sagemaker step operator** which will be utilized to run the training
   step on Sagemaker.
+* An **Image Builder** which will be used to build the docker image that will
```
Comment (strickvl) [inline examples/step_operator_remote_training/README.md:70]:
```suggestion
* An **Image Builder** which will be used to build the Docker image that will
```

File: examples/tekton_pipelines_orchestration/README.md [hunk 5]
```diff
@@ -120,6 +120,7 @@ GCP **container registry**.
 * The **Tekton orchestrator** is responsible for running your ZenML pipeline 
 in Tekton Pipelines. We need to configure it with the right kubernetes context 
 so ZenML can run pipelines in your GCP cluster. 
+* An **Image Builder** that builds docker images for your pipeline steps.
```
Comment (strickvl) [inline examples/tekton_pipelines_orchestration/README.md:123]:
```suggestion
* An **Image Builder** that builds Docker images for your pipeline steps.
```

**Review by strickvl — CHANGES_REQUESTED** [review]

**Review by strickvl — APPROVED** [review]
Looking good now. I guess we need to eventually do this to all projects, zenbytes, blog, and within the docs as well?

**Comment by safoinme** [comment]
Yes i should apply same changes to other repos

**Pull request merged**